data.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
We seem to have two copies of `data.json` now. Since, as we discussed, we'd really prefer just to ship a single binary, not a binary plus a `data.json` file, we could use the `embed` library to embed this data into the binary! (We'd likely embed this directly as a `map[string]Question`, rather than a JSON string.)

See https://www.youtube.com/watch?v=9HiVYGOMLiE

Reply:
This is an interesting feature. I tried to embed directly to map[string]Question but that gave me an error that it cannot embed to map[string]Question and found this reference - https://blog.carlmjohnson.net/post/2021/how-to-use-go-embed/  which has this section - Embedding complex structs and so am now reading the file into [] byte and then unmarshalling it. 
Thanks for this suggestion - very useful and I no longer need to pass the file name as an argument to NewGame.